pkg/cedar/core/authority_to_operate.go

```diff
@@ -0,0 +1,126 @@
+package cedarcore
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/guregu/null"
+	"github.com/guregu/null/zero"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/models"
+
+	apiauthority "github.com/cmsgov/easi-app/pkg/cedar/core/gen/client/authority_to_operate"
+)
+
+// GetAuthorityToOperateOptionalParams represents the optional parameters that can be used to filter ATO information when searching through the CEDAR API
+type GetAuthorityToOperateOptionalParams struct {
```

Comment:
As discussed in the CEDAR meeting and on Slack, we should be able to remove this, and just use `cedarSystemID` as a mandatory parameter.

Reply:
Done as of https://github.com/CMSgov/easi-app/pull/1636/commits/b421c6bbac17cdbb5fb9086448d5bcce549eb523.